### PR TITLE
(chore): cleanup ado, add badge

### PR DESCRIPTION
**Pull request by schristoff**

### CODE REVIEW

File: .github/workflows/porter-install-check.yml [hunk 1]
```diff
@@ -25,8 +25,7 @@ jobs:
       shell: powershell
   macos:
     runs-on:
-      - self-hosted
-      - macOS-latest
+      - macos-latest
```
Comment (troy0820) [inline .github/workflows/porter-install-check.yml:28]: Did we want to do this on Windows? I know that @ludfjig  was having issues with this at times. If it's easier not to, we don't have to.

File: build/testdata/bundles/mysql/.cnab/bundle.json
Comment (troy0820) [inline build/testdata/bundles/mysql/.cnab/bundle.json:1]: This is empty?  Can we delete this?

File: .github/workflows/build_azure_pipelinesrelease_template.yml [hunk 2]
```diff
@@ -78,7 +78,7 @@ jobs:
   Validate-unit_test:
     name: Unit Test
     runs-on: ubuntu-latest
-    if: !(inputs.skipTests)
+    if: ${{inputs.skipTests == false}}
```
Comment (troy0820) [inline .github/workflows/build_azure_pipelinesrelease_template.yml:81]: Only thing I would say is if we are moving from Azure pipelines, maybe let's rename the  file?

File: .github/workflows/porter-install-check.yml (hunk 1 above)
Comment (troy0820) [reply .github/workflows/porter-install-check.yml:28]: Nevermind, I see we already do this.

**Review by troy0820 — APPROVED** [review]
This LGTM.  The rename of the file is a nit and it's non-blocking.

File: build/testdata/bundles/mysql/.cnab/bundle.json
Comment (schristoff) [reply build/testdata/bundles/mysql/.cnab/bundle.json:1]: It gets made when I delete files it expects :/

**Pull request merged**